NewEmergmed/sackett/widgets.py

```diff
@@ -19,24 +19,24 @@ def __init__(self, searchable=False, fill_parent=True, default_text=None, index_
         self.default_text = default_text
         self.index_items = index_items
 
-    def render(self, name, value, attrs=None, choices=()):
+    def render(self, name, value, attrs=None, choices=(), renderer=None):
+        print(attrs.keys())
         if value is None:
             value = ''
-        final_attrs = self.build_attrs(attrs, name=name)
+        final_attrs = self.build_attrs(self.attrs if 'id' not in attrs.keys() else attrs, {'name': name})
```

Comment:
Let's talk through this one so I understand what the problem was and how this resolves it